Strats/Greek-Mean-Reversion/Ms_gmr_strat.py

```diff
@@ -0,0 +1,73 @@
+import math
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.linear_model import LinearRegression
+import Ss_gmr_strat
+
+class Ms_gmr_strat:
+    
+    # Inputs: kwargs with lists of data
+    # ** kwargs schema:
+    # kwargs = {
+    #     "tickers": ["AAPL"],    <- Lowercase only
+    #     "data": [**kwargs],
+    # }
+    
+    def __init__(self):
+        
+        # Inputs: **kwargs with a list of **kwargs
+        self.ticker_data = {}
+        self.decisions = {}
+        
+    
+    
+    ####################################################################
+    # GETTER: Get the decisions of a particular ticker
+    ####################################################################
+    def get_decisions(self, ticker, option_type, strategy, gORv):
+        
+        gamma_decisions, vega_decisions = self.ticker_data[ticker].get_gamma_vega_decisions()
+        
+        if (gORv == "gamma"):
+            return gamma_decisions[option_type][strategy]
+        elif (gORv == "vega"):
+            return vega_decisions[option_type][strategy]
+        else:
+            raise ValueError("gORv must be 'gamma' or 'vega'")
```

Comment:
we'd rather not raise errors - just print an error message for now, the linkers will take care of logs and whatnot